### ISSUE TITLE
v0-compatiable access of data field

### ISSUE THREAD

**Issue by pelikhan** [Tinkertanker/pxt-oled-ssd1306#12]
Use this macro to access the ``data`` field in v1 and v0 safely.

```
// v0 backward compat support
#ifndef PXT_BUFFER_DATA
#define PXT_BUFFER_DATA(buffer) buffer->payload
#endif
```

**Comment by nioq** [comment]
Good to know this exists. v0 compatibility was a bit worrying. Thanks!
Fixed in 53fe6a7

**Comment by pelikhan** [comment]
We are also locking the version using v0 so that you can safely use new features in the future. In particular it seems that we've snapped to ``v1.2.3`` but I can't find a release for it - https://github.com/Microsoft/pxt-microbit/blob/v0/targetconfig.json#L63 . Should we update it to this release?
